src/controllers/usersController.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-console */
+const userService = require('../services/usersServices');
+
+const get = async (req, res) => {
+  const users = await userService
+    .getAll()
+    .then((usrs) => usrs.map(userService.normalize));
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const user = userService.normalize(req.entry);
+
+  res.send(user);
+};
+
+const create = async (req, res) => {
+  const { name } = req.body;
+
+  const user = await userService.create(name).then(userService.normalize);
```

Comment:
Inconsistent promise handling: Consider using `async/await` instead of mixing it with `.then()`. This will make the code more readable and maintainable.